modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/model/IdentityProviderPOSTRequest.java

```diff
@@ -0,0 +1,400 @@
+    }
+
+    /**
+     * Convert the given object to string with each line indented by 4 spaces
+     * (except the first line).
+     */
+    private static String toIndentedString(java.lang.Object o) {
+
+        if (o == null) {
+            return "null";
+        }
+        return o.toString();
+    }
+
+    public static class Certificate {
+        private List<String> certificates = null;
+        private String jwksUri;
+
+        /**
+         *
+         **/
+        public Certificate certificates(List<String> certificates) {
+
+            this.certificates = certificates;
+            return this;
+        }
+
+        @ApiModelProperty()
+        @JsonProperty("certificates")
+        @Valid
+        public List<String> getCertificates() {
+            return certificates;
+        }
+
+        public void setCertificates(List<String> certificates) {
+            this.certificates = certificates;
+        }
+
+        public Certificate addCertificates(String certificate) {
+            if (this.certificates == null) {
+                this.certificates = new ArrayList<>();
+            }
+            this.certificates.add(certificate);
+            return this;
+        }
+
+        /**
+         *
+         **/
+        public Certificate jwksUri(String jwksUri) {
+
+            this.jwksUri = jwksUri;
+            return this;
+        }
+
+        @ApiModelProperty()
+        @JsonProperty("jwksUri")
+        @Valid
+        public String getJwksUri() {
+            return jwksUri;
+        }
+
+        public void setJwksUri(String jwksUri) {
+            this.jwksUri = jwksUri;
+        }
+
+        @Override
+        public boolean equals(Object o) {
+
+            if (this == o) {
+                return true;
+            }
+            if (o == null || getClass() != o.getClass()) {
+                return false;
+            }
+            Certificate certificate = (Certificate) o;
+            return Objects.equals(this.certificates, certificate.certificates) &&
+                    Objects.equals(this.jwksUri, certificate.jwksUri);
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hash(certificates, jwksUri);
+        }
+
+        @Override
+        public String toString() {
+
+            return "class Certificate {\n" +
+                    "    certificates: " + toIndentedString(certificates) + "\n" +
+                    "    jwksUri: " + toIndentedString(jwksUri) + "\n" +
+                    "}";
+        }
+    }
+}
```

Comment:
add new line

Reply:
Fixed with https://github.com/wso2/product-is/pull/16271/commits/e9343598362e5021e1f6832b61ef1997495afeda